backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala

```diff
@@ -294,6 +297,14 @@ object VeloxConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val COLUMNAR_VELOX_RESIZE_BATCHES_SHUFFLE_OUTPUT =
+    buildConf("spark.gluten.sql.columnar.backend.velox.resizeBatches.shuffleOutput")
+      .internal()
+      .doc(s"If true, combine small columnar batches together right after shuffle read. " +
+        s"The default minimum output batch size is equal to 0.8 * ${COLUMNAR_MAX_BATCH_SIZE.key}")
```

Comment:
Is 0.8 out-of-date? I see it's now 0.25. Can you update it in passing?

Reply:
ok

Reply:
done